specdb/specdb.py

```diff
@@ -364,9 +364,11 @@ def spectra_from_meta(self, meta, debug=False):
         sv_rows = []
         for group in groups:
             sub_meta = meta['GROUP'] == group
-            sv_rows.append(np.where(sub_meta)[0])
             # Grab
-            all_spec.append(self[group].spec_from_meta(meta[sub_meta]))
+            spec = self[group].spec_from_meta(meta[sub_meta])
+            if spec is not None:
+                all_spec.append(spec)
```

Comment:
If no spectrum is returned, then the return 'spec' object will not be aligned with the input meta table as stated in the docstring, right?  Not sure the best way to handle this, but at the very least we could throw a warning.  Or, we could throw a warning and return an array of indices into meta corresponding to the spectra we found.

Reply:
good question, but because the sv_rows line
is in this if statement, the two remain aligned.

Reply:
Good question, but because the sv_rows line is in this
if statement, the two remain aligned.  Have tested it.